sycl/test/sub_group/generic-shuffle.cpp

```diff
@@ -0,0 +1,214 @@
+// UNSUPPORTED: cuda
+// CUDA compilation and runtime do not yet support sub-groups.
+//
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUNx: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+//
+//==-- generic_shuffle.cpp - SYCL sub_group generic shuffle test *- C++ -*--==//
```

Comment:
There is no need in copyright header if test was written by you

Reply:
I based the test on `shuffle.cpp` in the same directory, which was written by @vladimirlaz and includes the copyright header.  Does that mean I should keep it?

Reply:
I don't know for sure. @bader?

Reply:
I suggest applying this comment to all SYCL LIT test in a separate PR.